action.yaml

```diff
@@ -46,26 +52,52 @@ inputs:
     description: 'Secret name to pull env variables from AWS Secret Manager'
     required: false
     default: ''
+  
+  # Application
+  app_directory:
+    description: 'Relative path for the directory of the app (i.e. where `Dockerfile` and `docker-compose.yaml` files are located). This is the directory that is copied to the EC2 instance.  Default is the root of the repo.'
   app_port:
     description: 'Port to expose for the app'
     required: false
-    default: '3000'
```

Comment:
why was this removed?

Reply:
All defaults have been moved as close to terraform as possible. In this case, the default is handled in the variables.tf file

Reply:
Defaults that could be have been moved as close to the terraform variables.tf file as possible. In this case, the default has been moved exactly there.

Reply:
right, but why?